src/schema/v2/CollectorProfile/helpers/formatName.ts

```diff
@@ -0,0 +1,33 @@
+type NameFormatType = "default" | "public"
```

Comment:
nit: no need for the suffix `Type` for TS types/interfaces, just use the normal pascal case.

Reply:
good point!